Examples/Examples/FeatureFormExampleView.swift

```diff
@@ -317,55 +325,26 @@ class Model: ObservableObject {
         if resultErrors.isEmpty {
             state = .idle
         } else {
-            // Additionally, you could display the errors to the user using `resultErrors`.
-            state = .generalError(featureForm, Text("Changes were not applied."))
+            state = .generalError(featureForm, Text("Apply edits failed with ^[\(resultErrors.count) error](inflect: true)."))
```

Comment:
Since we don't have other inline comments in this example code, I felt it was better to forego the comment and consume the errors in this basic way. Is this okay?